esp/src/src-react/components/GroupMembers.tsx

```diff
@@ -0,0 +1,146 @@
+import * as React from "react";
+import { CommandBar, ContextualMenuItemType, ICommandBarItemProps } from "@fluentui/react";
+import { useConst } from "@fluentui/react-hooks";
+import { scopedLogger } from "@hpcc-js/util";
+import * as Observable from "dojo/store/Observable";
+import { Memory } from "src/Memory";
+import * as WsAccess from "src/ws_access";
+import nlsHPCC from "src/nlsHPCC";
+import { ShortVerticalDivider } from "./Common";
+import { pushUrl } from "src-react/util/history";
+import { HolyGrail } from "src-react/layouts/HolyGrail";
```

Comment:
`src-react` -> `..`  (this is a bit pedantic I know, I suspect we can start ignoring them at some stage?)